code/mob/living/critter/bear.dm

```diff
@@ -1,23 +1,56 @@
 /mob/living/critter/bear
 	name = "space bear"
 	real_name = "space bear"
-	desc = "Oh god."
-	density = 1
```

Comment:
is removing the density intentional? is that set somewhere else or can you just walk through bears now

Reply:
Density is TRUE on the critter parent